### PR TITLE
feat: add JWT.getCredentials method

### LINKED ISSUE #313: JWT: Can I call `getCredentials`?
The JWTClient doesn't appear to let me call a "getCredentials()" function. Ideally, the JWTClient could recognize the p12/pem file, parse it, and internally store then return the private key from `getCredentials()`.

### PR DESCRIPTION
**Pull request by JustinBeckwith**
Resolves #313

### CODE REVIEW

File: src/auth/jwtclient.ts [hunk 1]
```diff
@@ -168,6 +168,20 @@ export class JWT extends OAuth2Client {
    * @private
    */
   async refreshToken(refreshToken?: string|null): Promise<GetTokenResponse> {
+    this.createGToken();
+    const token = await this.gtoken!.getToken();
```
Comment (jinwoo) [inline src/auth/jwtclient.ts]: What about making `createGToken()` return a string? Then you won't need the non-null annotation here.

File: src/auth/jwtclient.ts [hunk 2]
```diff
@@ -261,4 +268,19 @@ export class JWT extends OAuth2Client {
     }
     this.apiKey = apiKey;
   }
+
+  /**
+   * Using the key or keyFile on the JWT client, obtain an object that contains
+   * the key and the client email.
+   */
+  async getCredentials() {
```
Comment (jinwoo) [inline src/auth/jwtclient.ts]: I'd like public methods to have their return type (and param types) explicitly specified.

Comment (JustinBeckwith) [reply src/auth/jwtclient.ts]: 👍

File: src/auth/jwtclient.ts (hunk 1 above)
Comment (JustinBeckwith) [reply src/auth/jwtclient.ts]: 👍

Comment (jinwoo) [reply src/auth/jwtclient.ts]: Not done here and in line 177 below ?

Comment (JustinBeckwith) [reply src/auth/jwtclient.ts]: Ooops, sorry bout that.  Cleared it up.

**Pull request merged**